tests/wikidata/test_extract.py

```diff
@@ -18,12 +18,20 @@
 @pytest.mark.integration
 def test_search_organization_by_label() -> None:
     """Test organization search in wikidata."""
-    expected = "Q26678"
+    expected = "Q679041"
 
-    search_results = list(search_organization_by_label(item_label="BMW"))
+    search_results = search_organization_by_label(item_label='Robert Koch Institute"')
+    if search_results:
+        assert search_results.identifier == expected
+    else:
+        pytest.fail(f"No organizations were found for id: {expected}")
```

Comment:
why pytest.fail, and not simply `assert search_result`?

Reply:
I was trying to be very explicit. 😩

Reply:
ok, that is always good of course ;) just the `pytest.fail` syntax is not very intuitive. if you'd like to provide a custom error message to an assertion you can still do that with the `assert` syntax. just add a `,` and your message:

```python
assert search_results, f"No organizations were found for id: {expected}"
assert search_results.identifier == expected
```

Reply:
this is neater. I like it. implemented.